django_setup_configuration/test_utils.py

```diff
@@ -0,0 +1,11 @@
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.runner import SetupConfigurationRunner
+
+
+def load_step_config_from_source(
+    step: BaseConfigurationStep, yaml_file_path: str | None = None
```

Comment:
I seem to be getting typing errors here:

```python
def get_config_model(path):
    return load_step_config_from_source(ObjectsAPIConfigurationStep, path)
```


```Argument of type "type[ObjectsAPIConfigurationStep]" cannot be assigned to parameter "step" of type "BaseConfigurationStep[Unknown]" in function "load_step_config_from_source"
  "type[ABCMeta]" is not assignable to "type[BaseConfigurationStep[Unknown]]
```

Similarly, when using `step.execute(model)`, I also get typing errors:

```
Argument of type "ConfigurationModel" cannot be assigned to parameter "model" of type "ObjectsAPIGroupConfigModel" in function "execute"
  "ConfigurationModel" is not assignable to "ObjectsAPIGroupConfigModel"
```

Reply:
No longer getting errors :+1: